Packs/CommonPlaybooks/Playbooks/playbook-Containment_Plan_-_Disable_Account.yml

```diff
@@ -0,0 +1,281 @@
+    taskid: 073f607c-a924-43c1-8852-d480e1454fdc
+    type: condition
+    task:
+      id: 073f607c-a924-43c1-8852-d480e1454fdc
+      version: -1
+      name: Should disable the account?
+      description: Whether to disable the account based on the input values.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "2"
+      "yes":
+      - "13"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.UserContainment
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: inputs.Username
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 0
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "13":
+    id: "13"
+    taskid: 32ec3207-df33-4d5f-8551-2a49ca677b62
+    type: regular
+    task:
+      id: 32ec3207-df33-4d5f-8551-2a49ca677b62
+      version: -1
+      name: Set users to disable
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.11/Cortex-XSOAR-Administrator-Guide/Automations
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "18"
+    scriptarguments:
+      key:
+        simple: UsersToDisable
+      value:
+        complex:
+          root: inputs.Username
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 210
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "18":
+    id: "18"
+    taskid: b35ee9bb-51f1-4f64-8df5-b490d1592b2c
+    type: playbook
+    task:
+      id: b35ee9bb-51f1-4f64-8df5-b490d1592b2c
+      version: -1
+      name: Block Account - Generic v2
+      playbookName: Block Account - Generic v2
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "8"
+    scriptarguments:
+      Tag:
+        simple: Bad Account
```

Comment:
I think we should let the user tag it, or tag it with some kind of information that could point back to the incident. something like:

XSIAM - ${alert.id}

Reply:
The following tag was added: XSIAM alert - ${alert.id}